planner/utils/google_utils.py

```diff
@@ -0,0 +1,79 @@
+"""
+Functions for getting attributes about points of interest.
+Extracts data from POI locations (elevation, safety, etc.)
+
+Uses: Google Place Search
+
+Uses the python-google-places wrapper:
+https://github.com/slimkrazy/python-google-places
+"""
+
+from googleplaces import GooglePlaces, types
+from configparser import ConfigParser
+import psycopg2
+import json
+
+class GoogleHelper:
+	"""
+	Helper class for accessing the google API
+	"""
+	def __init__(self, gmaps_api_key=None, conn=None):
+		import db_conn
+		if gmaps_api_key is None:
+			config = ConfigParser()
```

Comment:
`ConfigParser` here is unused. If it's unneeded we can remove it and the import on Line 12 too.

Reply:
Oops - I'm going to just take all the db stuff out in this file because it's not being used as of now